pkg/db/advisory_locks.go

```diff
@@ -103,7 +103,6 @@ func (f *AdvisoryLockFactory) NewAdvisoryLock(ctx context.Context, id string, lo
 		return *lock.uuid, fmt.Errorf(errMsg)
 	}
 
-	log.Debugf("Locked advisory lock id=%s type=%s - owner=%s", id, lockType, *lock.uuid)
```

Comment:
maybe change the log level to trace instead of removing this?

Reply:
If an error occurs, it is captured and returned. If there is no error, it means the lock was successfully acquired. Do you really need to know the detailed information from this log?

Reply:
these locks are released by transaction，so there may be no errors, keeping this part will help us to check the lock status if some accident problems occurs on lock part

Reply:
And by default, we may use info level for maestro server logs, not the debug?

Reply:
in ARO HCP, they are using debug by default.

Reply:
Discussed with @skeeey offline, we agree that expose the lock/unlock count in metrics can address the current concern. Will submit another PR to refine the metrics to reflect on this idea.